[2024-09-12T12:57:38.242-0300] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-12T12:57:38.249-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_test.update_data manual__2024-09-12T15:57:35.291800+00:00 [queued]>
[2024-09-12T12:57:38.254-0300] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_test.update_data manual__2024-09-12T15:57:35.291800+00:00 [queued]>
[2024-09-12T12:57:38.254-0300] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-12T12:57:38.266-0300] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): update_data> on 2024-09-12 15:57:35.291800+00:00
[2024-09-12T12:57:38.269-0300] {standard_task_runner.py:72} INFO - Started process 9569 to run task
[2024-09-12T12:57:38.271-0300] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'dag_test', 'update_data', 'manual__2024-09-12T15:57:35.291800+00:00', '--job-id', '37', '--raw', '--subdir', 'DAGS_FOLDER/dag_test.py', '--cfg-path', '/tmp/tmpiefh2yhk']
[2024-09-12T12:57:38.272-0300] {standard_task_runner.py:105} INFO - Job 37: Subtask update_data
[2024-09-12T12:57:38.296-0300] {task_command.py:467} INFO - Running <TaskInstance: dag_test.update_data manual__2024-09-12T15:57:35.291800+00:00 [running]> on host DESKTOP-PEJJLJO.
[2024-09-12T12:57:38.344-0300] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='dag_test' AIRFLOW_CTX_TASK_ID='update_data' AIRFLOW_CTX_EXECUTION_DATE='2024-09-12T15:57:35.291800+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-12T15:57:35.291800+00:00'
[2024-09-12T12:57:38.345-0300] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-12T12:57:38.356-0300] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2024-09-12T12:57:38.357-0300] {base.py:84} INFO - Retrieving connection 'google_cloud_default'
[2024-09-12T12:57:39.474-0300] {dag_test.py:78} INFO - {'spreadsheetId': '1EGfO6zUlbZqMfXmMU7_CXh4PpYpQTFgCjyVuqM1fT3I', 'updatedRange': "'PÃ¡gina1'!A1:C1", 'updatedRows': 1, 'updatedColumns': 3, 'updatedCells': 3}
[2024-09-12T12:57:39.474-0300] {python.py:240} INFO - Done. Returned value was: None
[2024-09-12T12:57:39.477-0300] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-12T12:57:39.477-0300] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_test, task_id=update_data, run_id=manual__2024-09-12T15:57:35.291800+00:00, execution_date=20240912T155735, start_date=20240912T155738, end_date=20240912T155739
[2024-09-12T12:57:39.527-0300] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-12T12:57:39.538-0300] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-12T12:57:39.543-0300] {local_task_job_runner.py:245} INFO - ::endgroup::
